packages/bpk-component-graphic-promo/src/BpkGraphicPromo-test.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import { cssModules } from 'bpk-react-utils';
+
+import BpkGraphicPromo from './BpkGraphicPromo';
+import TEXT_ALIGN from './BpkGraphicPromoTextAlign';
+import STYLES from './BpkGraphicPromo.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+const props = {
+  kicker: 'Kicker',
+  headline: 'Ride your wave',
+  strapline:
+    'Portugal and 6 more countries have just been added to the UK travel green list',
+  image: 'path/to/image.jpg',
+  sponsor: {
+    label: 'Sponsored',
+    logo: 'path/to/logo.png',
+    altText: 'Airline Name',
+  },
+  ctaText: 'Learn more',
+  onClick: jest.fn(),
+  textAlign: TEXT_ALIGN.start,
+  invertVertically: false,
+};
+
+describe('BpkGraphicPromo', () => {
+  beforeEach(() => {
+    props.onClick.mockReset();
+  });
+
+  it('should render correctly with all properties set', () => {
+    const { asFragment } = render(<BpkGraphicPromo {...props} />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should support custom class names', () => {
+    const { asFragment } = render(
+      <BpkGraphicPromo className="custom-classname" {...props} />,
+    );
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should support arbitrary props', () => {
+    const { asFragment } = render(<BpkGraphicPromo testid="123" {...props} />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should render correctly when centre aligned', async () => {
+    const customProps = { ...props, textAlign: 'center' };
+    const { asFragment } = render(<BpkGraphicPromo {...customProps} />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should render correctly when right aligned', async () => {
```

Comment:
are the asyncs in this and the following tests required?

Reply:
Good catch! Removing these right away. 🙂